dccrg.hpp

```diff
@@ -8759,13 +8791,27 @@ template <
 		}
 		#endif
 
+		this->user_neigh_of[neighborhood_id][cell].clear();
+		this->user_neigh_to[neighborhood_id][cell].clear();
+		if (this->cell_process.count(cell) == 0) {
+			return;
+		}
+		if (!allowRemotes) {
+			if (this->cell_process.at(cell) != this->rank) {
+				return;
+			}
+		}
+		if (cell != this->get_child(cell)) {
+			return;
+		}
+
```

Comment:
I was about to say "this clear is kinda unnecessary, since the value is being completely reassigned afterwards anyway".
But I assume there are some good reasons *these specific* returns are in there, when cells have been refined away and actually need to be cleared here.